docs/changelog.md

```diff
@@ -5,6 +5,9 @@ Lines for version numbers should always be formatted as
 with nothing else on the line.
 -->
 * HEAD
+* [0.22.1](https://pypi.org/project/sematic/0.22.1/)
+    * [bugfix] To the script to allow auto-upgrading settings files.
+      **IMPORTANT**: Users should run `sematic migrate` upon install.
```

Comment:
Please mention the hidden run UI feature and the minor improvements to the backend API,